pulp_ansible/app/galaxy/v3/serializers.py

```diff
@@ -235,6 +261,13 @@ def get_git_commit_sha(self, obj) -> str:
         if not content_artifact.get().artifact:
             return content_artifact.get().remoteartifact_set.all()[0].url[-40:]
 
+    def get_signatures(self, obj):
+        """
+        Get the signatures.
+        """
+        filtered_signatures = obj.signatures.filter(pk__in=self.context["sigs"])
```

Comment:
This is a bit fuzzy to me. If i get it correctly, this only works when the collection version is listed as part of a distribution.
What if we did (assuming that passing `None` to `pk__in` is like not passing at all [with python you never _know_]):
```suggestion
        filtered_signatures = obj.signatures.filter(pk__in=self.context.get("sigs"))
```

Then again, this is used in the galaxy api, and there may always be that `_distro_content`.

Reply:
I tried passing in None, django gives an error. I'll make the default an empty list, that should return an empty queryset.